cps/opds.py

```diff
@@ -326,12 +326,12 @@ def feed_shelfindex():
 def feed_shelf(book_id):
     off = request.args.get("offset") or 0
     if current_user.is_anonymous:
-        shelf = ub.session.query(ub.Shelf).filter(ub.Shelf.is_public == 1, ub.Shelf.id == book_id).first()
+        shelf = ub.session.query(ub.Shelf).filter(ub.Shelf.is_public == 1, ub.Shelf.id == book_id, not ub.Shelf.deleted).first()
     else:
```

Comment:
We almost got it:
![grafik](https://user-images.githubusercontent.com/6438895/80298541-a95ffc80-878d-11ea-821f-6a2a4fb503e4.png)
But there is one sideeffect: This line and line 334 causes a crash upon access. Because there is no column deleted in Shelf. I currently can't oversee what the intention of the change was. I would delete it?